lib/stylesheet/manager.rb

```diff
@@ -135,7 +135,9 @@ def self.max_file_mtime
   end
 
   def self.cache_fullpath
-    "#{Rails.root}/#{CACHE_PATH}"
+    path = "#{Rails.root}/#{CACHE_PATH}"
```

Comment:
Prior art for a fix like this:

https://github.com/discourse/discourse/blob/599f3784452e1618d25ff962203d1abfd3383f0c/lib/file_store/base_store.rb#L37-L41